staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -106,7 +106,18 @@ import (
 // and index updates happen before such a prescribed notification.
 // For a given SharedInformer and client, the notifications are
 // delivered sequentially.  For a given SharedInformer, client, and
-// object ID, the notifications are delivered in order.
+// object ID, the notifications are delivered in order.  Because
+// `ObjectMeta.UID` has no role in identifying objects, if an object
```

Comment:
This is a very long sentence and it has two commas.  When a sentence has two commas, they should set off a parenthetic remark --- but that is not what is happening here.  I suggest rewording, something like the following.

> Because `ObjectMeta.UID` has no role in identifying objects, it is possible that when (1) object O1 with ID (e.g. namespace and name) X and `ObjectMeta.UID` U1 in the SharedInformer's local cache is deleted and later (2) another object O2 with ID X and `ObjectMeta.UID` U2 is created the informer's clients are not notified of (1) and (2) but rather are notified only of an update from O1 to O2.

Reply:
Reworded as you suggested in 21f59c9.